app.py

```diff
@@ -0,0 +1,88 @@
+from os import read, write
+from flask import Flask, request, jsonify, render_template
+import flask
+import json
+from jsonschema import validate
+import jsonschema
+from flask_ngrok import run_with_ngrok
+
+
+
+app = Flask(__name__)
+app.config["DEBUG"] = True
+# run_with_ngrok(app)
+
+#Loading schema to a variable
+schema = json.load(open('schema.json'))
+
+#Reading users data from json file and storing them as variable
+json_object = json.load(open('users.json'))
```

Comment:
It's fine for a start, but don't use global variables. For the next iteration add helper functions that will read/write the JSON to a file and execute them from the endpoint handler.